src/index.ts

```diff
@@ -44,6 +44,11 @@ if (config.health.healthz.enabled) {
     Healthz.listen();
 }
 
+async function createManagementRoom(client: MatrixClient, alias: string) {
```

Comment:
Could you document this? What is it for? Also, it might be useful to return the room id.